engine/model_draft_transactions_test.go

```diff
@@ -284,18 +292,18 @@ func TestDraftTransaction_createTransaction(t *testing.T) {
 
 		assert.Equal(t, testXPubID, draftTransaction.Configuration.ChangeDestinations[0].XpubID)
 		assert.Equal(t, draftTransaction.ID, draftTransaction.Configuration.ChangeDestinations[0].DraftID)
-		assert.Equal(t, uint64(98988), draftTransaction.Configuration.ChangeSatoshis)
+		assert.Equal(t, uint64((startingBalance - txAmount - expectedFee)), draftTransaction.Configuration.ChangeSatoshis)
 
-		assert.Equal(t, uint64(12), draftTransaction.Configuration.Fee)
+		assert.Equal(t, uint64(expectedFee), draftTransaction.Configuration.Fee)
 		assert.Equal(t, *chainstate.MockDefaultFee, *draftTransaction.Configuration.FeeUnit)
 
 		assert.Equal(t, 1, len(draftTransaction.Configuration.Inputs))
 		assert.Equal(t, testLockingScript, draftTransaction.Configuration.Inputs[0].ScriptPubKey)
-		assert.Equal(t, uint64(100000), draftTransaction.Configuration.Inputs[0].Satoshis)
+		assert.Equal(t, uint64(startingBalance), draftTransaction.Configuration.Inputs[0].Satoshis)
 
 		assert.Equal(t, 2, len(draftTransaction.Configuration.Outputs))
-		assert.Equal(t, uint64(1000), draftTransaction.Configuration.Outputs[0].Satoshis)
-		assert.Equal(t, uint64(98988), draftTransaction.Configuration.Outputs[1].Satoshis)
+		assert.Equal(t, uint64(txAmount), draftTransaction.Configuration.Outputs[0].Satoshis)
+		assert.Equal(t, uint64((startingBalance - txAmount - expectedFee)), draftTransaction.Configuration.Outputs[1].Satoshis)
```

Comment:
i would say use `require` package instead of `assert`

Reply:
@ac4ch I agree to creating structs instead of json but maybe we can create separate task and discuss if we want to change to require and do this everywhere

Reply:
task for discussion created